docs/examples/plot_ghi_transposition.py

```diff
@@ -0,0 +1,82 @@
+"""
+GHI to POA Transposition
+=========================
+
+Example of generating clearsky GHI and POA irradiance.
+"""
+
+# %%
+# This example shows how to use the
+# :py:meth:`pvlib.location.Location.get_clearsky` method to generate clearsky
+# GHI data as well as how to use the
+# :py:meth:`pvlib.irradiance.get_total_irradiance` function to transpose
+# GHI data to Plane of Array (POA) irradiance.
+
+from pvlib import location
+from pvlib import irradiance
+import pandas as pd
+from matplotlib import pyplot as plt
+
+# For this example, we will be using Golden, Colorado
+tz = 'MST'
+lat, lon = 39.755, -105.221
+
+# Create location object to store lat, lon, timezone
+site = location.Location(lat, lon, tz=tz)
+
+
+# Calculate clear-sky GHI and transpose to plane of array
+# Define a function so that we can re-use the sequence of operations with
+# Different locations
+def get_irradiance(site_location, date, tilt, surface_azimuth):
+    # Creates one day's worth of 10 min intervals
+    times = pd.date_range(date, freq='10min', periods=6*24, tz=tz)
+    # Generate clearsky data using the Ineichen model, which is the default
+    # The get_clearsky method returns a dataframe with values for GHI, DNI,
+    # and DHI
+    clearsky_ghi = site_location.get_clearsky(times)
```

Comment:
if `get_clearsky` returns a "dataframe with values for GHI, DNI, and DHI" then IMO maybe don't call it `clearsky_ghi`? How about just `clearsky` or `clearsky_irrad`?
```python
clearsky = site_location.get_clearsky(times)

# then below
dni = clearsky['dni']
ghi = clearsky['ghi']
dhi = clearsky['dhi']
```
what do you think?